src/twilio/twilio_data_mappings.bal

```diff
@@ -97,7 +133,6 @@ function mapJsonToAuthyOtpResponse(json jsonPayload) returns AuthyOtpResponse {
     AuthyOtpResponse authyOtpResponse = {};
     authyOtpResponse.message = jsonPayload.message.toString();
     authyOtpResponse.cellphone = jsonPayload.cellphone.toString();
-    authyOtpResponse.device = jsonPayload.device.toString();
```

Comment:
We do not receive device attribute with the JSON payload. Hence removed code for setting it from here. Only three fields are present. E.g., {"success":true, "message":"SMS token was sent", "cellphone":"+94-XXX-XXX-XX22"}.